src/main/kotlin/org/rucca/cheese/model/PatchTaskSubmissionReviewRequestDTO.kt

```
@@ -0,0 +1,21 @@
+ */
+data class PatchTaskSubmissionReviewRequestDTO(
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("accepted")
+    val accepted: kotlin.Boolean? = null,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("score")
+    val score: kotlin.Int? = null,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("comment")
+    val comment: kotlin.String? = null
+) {}
```

detekt

[warning] The class or object PatchTaskSubmissionReviewRequestDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)